dkist/io/asdf/converters/tiled_dataset.py

```diff
@@ -11,7 +14,19 @@ class TiledDatasetConverter(Converter):
     def from_yaml_tree(cls, node, tag, ctx):
         from dkist.dataset.tiled_dataset import TiledDataset
 
-        return TiledDataset(node["datasets"], node["inventory"])
+        tiled_dataset = TiledDataset(node["datasets"], node["inventory"])
```

Comment:
Just a drive-by comment from my phone, but could we have this be built on demand in a property on TiledDataset rather than here?

With this implemented like this, does it reflect slicing of the sub-datasets correctly?

Reply:
I'm not sure if it does, that was the next thing I was going to test. But probably not. Presumably the aim of doing it as a property would be to have that automatically? I expect that would be easy to implement.